pkg/database/gorm.go

```diff
@@ -47,22 +33,28 @@ func getDialectorFromConfig(config *Config, environment string) gorm.Dialector {
 		connectionString = testDriverName
 	}
 
-	return mysql.New(mysql.Config{
-		DSN:        connectionString,
-		DriverName: driverName,
-	})
-}
+	serviceName := fmt.Sprintf("%s-mysql", appName)
 
-func databasePoolSettings(gormDB *gorm.DB, config *Config) error {
-	db, err := gormDB.DB()
+	sqltrace.Register(driverName, &mysqldriver.MySQLDriver{}, sqltrace.WithServiceName(serviceName))
```

Comment:
Should we be using the same `service` name for both? 🤔

Reply:
I absolutely support the idea, both service names are now using application name 👍 
Used a separate commit.

Reply:
@fotos I misunderstood your message 😅 Now it's  changed `-orm` for the driver tracing.

Reply:
You misunderstood because my question wasn't well phrases.

I can see how it can be interpreted both ways. Apologies for the confusion. 😊